z-client-mu-plugins.php

```diff
@@ -47,4 +47,51 @@ function wpcom_vip_load_client_mu_plugins() {
 	}
 }
 
+function get_client_mu_plugins() {
```

Comment:
There's a bunch of shared code between this and the function where we actually load the plugins. I think we can split this off into two functions and re-use some of the logic:

1) `wpcom_vip_get_client_mu_plugins()` which would get and return the plugins list, but not load plugin data.
2) `wpcom_vip_get_client_mu_plugins_data()` which would call `wpcom_vip_get_client_mu_plugins()` and then loop through to load plugin data and return.